zebra/zebra_vxlan_if.c

```diff
@@ -493,8 +486,7 @@ static int zebra_vxlan_if_vni_entry_add(struct zebra_if *zif,
 }
 
 static int zebra_vxlan_if_add_update_vni(struct zebra_if *zif,
-					 struct zebra_vxlan_vni *vni,
-					 void *ctxt)
+					 struct zebra_vxlan_vni *vni, void *ctxt)
```

Comment:
Can you please revert this change? This will make the lines go more than 80 chars. Similar comment for other lines as well.

Reply:
Thanks Mobash, 
Corrected as per the suggestion.